app/src/main/java/org/mozilla/focus/utils/DebugUtils.java

```diff
@@ -0,0 +1,38 @@
+package org.mozilla.focus.utils;
+/* -*- Mode: Java; c-basic-offset: 4; tab-width: 4; indent-tabs-mode: nil; -*-
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+
+import android.content.Context;
+import android.text.TextUtils;
+import android.webkit.WebView;
+
+public final class DebugUtils {
+
+    private DebugUtils() {
+
+    }
+
+    public static String loadWebViewVersion(Context context) {
```

Comment:
I'm curious of the reason why WebSettings.getDefaultUserAgent(context) is not used directly here.

Reply:
I think you're right. Using getDefaultUserAgent will make this more simple.

Reply:
There's a UnitTest fail in WebSettings.getDefaultUserAgent. I will rollback to use the original plan for now.